crossing-training.lic

```diff
@@ -457,27 +457,15 @@ class CrossingTraining
 
     if data['moon']
       # Look for a moon that will be up for the next few minutes
-      unless Script.running? 'moonwatch'
-        echo 'moonwatch is not running. Starting it now'
-        UserVars.moons = {}
-        custom_require.call('moonwatch')
-        echo "Run `;e autostart('moonwatch')` to avoid this in the future"
-        pause 0.5 while UserVars.moons.empty?
-      end
-      unless moon = UserVars.moons.find { |moon_name, moon_data| UserVars.moons['visible'].include?(moon_name) && moon_data['timer'] >= 4 }.first
-        echo "No moon available to cast #{data['abbrev']}"
-
+      check_moonwatch
+      unless set_moon_data(data)
         weather = bput('weather', 'inside', 'You glance up at the sky.')
```

Comment:
I was wondering if this was old code; since moonwatch updates the moon data in the background, setting the moon data isn't contingent on being inside/outside